apps/dashboard/app/models/announcement.rb

```diff
@@ -49,9 +49,15 @@ def opts
                 {}
               end
     @opts.symbolize_keys.compact
-  rescue Errno::ENOENT, SyntaxError # File does not exist or syntax errors
+  rescue Errno::ENOENT # File does not exist
+    Rails.logger.warn "Announcement file not found: #{@path}"
     @opts = {}
-  rescue StandardError => e
+  rescue SyntaxError => e # Syntax errors
+    @error = e.message
```

Comment:
Why memorize this member variable `@error`?  You don't appear to use it.

Reply:
Just removed!